lib/ansible/plugins/netconf/__init__.py

```diff
@@ -228,13 +242,6 @@ def commit(self, *args, **kwargs):
         resp = self.m.commit(*args, **kwargs)
         return resp.data_xml if hasattr(resp, 'data_xml') else resp.xml
 
-    @ensure_connected
```

Comment:
Why is this removed? It is used by other platforms based on Netconf.

Reply:
@ganeshrn 
The reason, I've removed `validate` is because it was there twice. Check around line 160... Don't believe there is specific reason to have the same function with identical content twice. Let me know, if I am mistaken.

Reply:
Good catch